networking/podenv.go

```diff
@@ -58,13 +58,13 @@ type activeNet struct {
 }
 
 // Loads nets specified by user and default one from stage1
-func (e *podEnv) loadNets() ([]activeNet, error) {
-	nets, err := loadUserNets()
+func (e *Networking) loadNets() ([]activeNet, error) {
+	nets, err := loadUserNets(e.netsLoadList)
```

Comment:
Can you just pass `netsLoadList` as an arg into `loadNets` or add `netsLoadList` as a member of `podEnv`, not `Networking`. `podenv.go` is for `podEnv` and its methods.